config/routes.rb

```diff
@@ -6,6 +6,8 @@
   get '/help/:page', :to => "help#page"
   get '/help', :to => "help#page", :page => "general"
 
+  get '/featured_researcher/:id', :to => "content_blocks#delete", :as => "delete_researcher"
```

Comment:
You'll want a Delete verb instead of Get if you're going directly to #delete. What you have is usually the route for show.

An example from Sufia's routes is:
```
delete 'users/:user_id/depositors/:id' => 'depositors#destroy', as: 'user_depositor'
```
at https://github.com/projecthydra/sufia/blob/master/config/routes.rb#L47